content/zh/docs/concepts/services-networking/dual-stack.md

```diff
@@ -159,7 +184,9 @@ If you would like to define which IP family to use for single stack or define th
 如果你想要定义哪个 IP 族用于单栈或定义双栈 IP 族的顺序，可以通过设置服务上的可选字段 `.spec.ipFamilies` 来选择地址族。
 
 <!--
+{{< note >}}
 The `.spec.ipFamilies` field is immutable because the `.spec.ClusterIP` cannot be reallocated on a Service that already exists. If you want to change `.spec.ipFamilies`, delete and recreate the Service.
+{{< /note >}}
```

Comment:
This kind of shortcode was intentionally removed because they sometimes still work when appearing between `<!--` and `-->`. So ... such fixes are not required and even undesirable.

Reply:
I will pay special attention to this.
Thank for your hint.